packages/esm-ohri-pmtct-app/src/form-entry/forms/labour-and-delivery/1.0.json

```diff
@@ -1116,7 +1116,10 @@
                   "id": "infantPtrackerid",
                   "questionOptions": {
                     "concept": "164803AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
-                    "rendering": "text"
+                    "rendering": "text",
+                    "calculate": {
+                      "calculateExpression": "MotherPtracker_id !== '6c45421e-2566-47cb-bbb3-07586fffbfe2' ? myValue = useFieldValue('MotherPtracker_id') : ''"
```

Comment:
I think you should check if mother's PTracker is empty rather than checking for that static id. Something like:
`!isEmpty('MotherPtracker_id') ? myValue = useFieldValue('MotherPtracker_id') : ''`

Reply:
@kajambiya Makes sense, changed